docs/configuration.md

```diff
@@ -0,0 +1,114 @@
+- [neon-audio](https://github.com/NeonGeckoCom/neon_audio)
+- [neon-skills](https://github.com/NeonGeckoCom/neoncore)
+- [neon-speech](https://github.com/NeonGeckoCom/neon_speech)
+- [neon-messagebus](https://github.com/NeonGeckoCom/neon_messagebus)
+
+## Common configuration options
+
+Common configurations are found in the `neon.yaml` file.
+
+!!! warning
+    YAML is a whitespace-sensitive language. Be sure to maintain proper indentation when editing the `neon.yaml` file.
+
+Specific configuration changes are also detailed in blog posts by community dev Mike Gray. [Read the blog posts](https://blog.graywind.org/tags/neon/).
+
+### Logging verbosity
+
+To increase or decrease the verbosity of the logs, change the `LOG_LEVEL` value. The default value is `INFO`.
+
+```yaml
+LOG_LEVEL: DEBUG
+```
+
+All valid Python logging levels are available. For reference, see the [Python logging documentation](https://docs.python.org/3/library/logging.html#logging-levels).
+
+### Units of measurement
+
+To switch between metric and imperial units, change the `system_unit` value. The default value is `imperial`.
+
+```yaml
+system_unit: metric
+```
+
+To switch between 12-hour and 24-hour time, change the `time_format` value. The default value is `half`, or 12-hour time.
+
+```yaml
+time_format: full
+```
+
+### Timezone
+
+To change the timezone manually, change the `location.timezone` value. The default value is `America/Los_Angeles`.
+
+```yaml
+location:
+  timezone: America/Chicago
+```
+
+## Installing additional skills and plugins
+
+Each Neon Core service is a Docker container. To install a new service, you need to add it to the `neon.yaml` file and then restart the services.
+
+For each package you want to install, add a [pip-compatible package name](https://pip.pypa.io/en/stable/reference/requirements-file-format/) to the `default_skills`, `extra_dependencies.voice`, `extra_dependencies.audio`, or `extra_dependencies.phal` lists in the `neon.yaml` file.
+
+Please note that the Python environments in each container do not persist past reboots, so you _must_ use the `neon.yaml` file to install new packages. Failure to do so will result in the package being removed when the container is restarted.
+
+Examples:
+
+```yaml
+default_skills: # Add skills to install here
+  - neon-skill-alerts
+  # - other pip-compatible package names
+extra_dependencies:
+  global: # Installs in all containers/services
+    - requests
+  skills: # Add neon-skill dependencies to install here
+    - neon-skill-alerts
+  voice: # Add neon-speech dependencies to install here
+    - ovos-stt-server
+  audio: # Add neon-audio dependencies to install here
+    - ovos-tts-plugin-beepspeak
+  enclosure: # Add neon-enclosure dependencies to install here
+    - ovos-PHAL-plugin-homeassistant
+```
+
+## External services
+
+The default `neon.yaml` file includes a commented section for using your own API keys for external services. It looks like this:
+
+```yaml
+# keys:
```

Comment:
The `keys` top-level key is no longer required (but is still supported) as of [this PR](https://github.com/NeonGeckoCom/neon_api_proxy/pull/99) (v 0.7.0)